### PR TITLE
Adapt Prerequisites in docs

### PR DESCRIPTION
**Pull request by wencke-lm**

### CODE REVIEW

File: docs/slurk_prerequisites.rst [hunk 1]
```diff
@@ -135,7 +144,7 @@ Checkout and run slurk
 
     $> ssh-keygen
 
-2. Upload or copy the generated public key to your github SSH settings
+2. Upload or copy the generated public key to your `github SSH settings <https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account>`_
```
Comment (TimDiekmann) [inline docs/slurk_prerequisites.rst:147]: Why is an ssh key required to run slurk? Why cloning is required at all? I'd just run the docker container with some environment variables and volumes passed. Maybe we want to split the section into "Run with docker (recommended)" and "Run without docker".

File: docs/slurk_prerequisites.rst [hunk 2]
```diff
@@ -197,4 +209,4 @@ This is discussed in the following issue:
 
 .. pull-quote::
     `microsoft/WSL#4903 (mdadm 4.1-2 causes unrecoverable grub config loop in dpkg, apt) <https://github.com/microsoft/WSL/issues/4903>`_
-    Spiritually similar #4763 (over there it was LVM). WSL2 does not load the Linux kernel with GRUB (and WSL1 doesn't load a Linux kernel at all). Linux RAID (mdadm) isn't applicable for lack of Multiple Disks. Somehow those were installed on your system. Purging those packages should help.
\ No newline at end of file
+    Spiritually similar #4763 (over there it was LVM). WSL2 does not load the Linux kernel with GRUB (and WSL1 doesn't load a Linux kernel at all). Linux RAID (mdadm) isn't applicable for lack of Multiple Disks. Somehow those were installed on your system. Purging those packages should help.
```
Comment (TimDiekmann) [inline docs/slurk_prerequisites.rst:212]:
```suggestion
    Spiritually similar #4763 (over there it was LVM). WSL2 does not load the Linux kernel with GRUB (and WSL1 doesn't load a Linux kernel at all). Linux RAID (mdadm) isn't applicable for the lack of Multiple Disks. Somehow those were installed on your system. Purging those packages should help.
```

File: docs/slurk_prerequisites.rst [hunk 3]
```diff
@@ -92,13 +101,13 @@ Install docker into Ubuntu
 
     $> curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
 
-4. Use the following command to set up the stable repository.
+4. Use the following command to set up the stable repository:
```
Comment (TimDiekmann) [inline docs/slurk_prerequisites.rst:104]: For installing docker better link to the [official documentation](https://docs.docker.com/get-docker/).

File: docs/slurk_prerequisites.rst [hunk 4]
```diff
@@ -92,13 +101,13 @@ Install docker into Ubuntu
...
 
 .. code-block:: bash
 
     $> sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
 
-5. Install the docker engine
+5. Install the docker engine:
```
Comment (TimDiekmann) [inline docs/slurk_prerequisites.rst:110]: Same as above.

File: docs/slurk_prerequisites.rst [hunk 5]
```diff
@@ -148,19 +157,22 @@ Checkout and run slurk
 
 .. code-block:: bash
 
-    $> source ./start_slurk_server.sh
-    $> echo $SLURK_SERVER_ID
-    ce96a2009c0be81f6a82161f611db2f5fd74ed95e8d9b8ea434aeb1bcbb2342b
+    $> export SLURK_DOCKER=slurky
+    $> bash scripts/start_server.sh
```
Comment (TimDiekmann) [inline docs/slurk_prerequisites.rst:161]: All scripts are marked as executable
```suggestion
    $ export SLURK_DOCKER=slurk
    $ scripts/start_server.sh
```

File: docs/slurk_prerequisites.rst [hunk 6]
```diff
@@ -148,19 +157,22 @@ Checkout and run slurk
...
+    b4ad4ccd053bebd4ce80afbeea3a3a259d890ae7103577e003cb2e4b768687fb
 
 6. Fetch the admin token
 
 .. code-block:: bash
 
-    $> source ./get_admin_token.sh
-    $> echo $ADMIN_TOKEN
-    d858f508-da42-4ffb-847c-c7f378c35e02
+    $> bash scripts/read_admin_token.sh
```
Comment (TimDiekmann) [inline docs/slurk_prerequisites.rst:168]:
```suggestion
    $ scripts/read_admin_token.sh
```

**Review by TimDiekmann — CHANGES_REQUESTED** [review]
Documentation for Windows 10 looks good although I can't test it right now.

Could you please replace `$> `  with `$ `?

File: docs/slurk_prerequisites.rst (hunk 1 above)
Comment (wencke-lm) [reply docs/slurk_prerequisites.rst:147]: You run the docker container with the provided script, so you would need that from the repository at least. I assume some would also want to add custom plugin scripts for which they may have to slightly alter slurk. I don't think having them clone slurk is a bad idea. I don't get the feeling that slurk is something you just use as it is.

Comment (TimDiekmann) [reply docs/slurk_prerequisites.rst:147]: I assume anyone who has write access to this repository knows how to push to git. For any other, `https` could be used:
```bash
git clone https://github.com/clp-research/slurk
```

**Review by TimDiekmann — APPROVED** [review]
Thanks!

**Pull request merged**